README.md

```diff
@@ -1,31 +1,15 @@
 # Gazebo ROS Demos
 
-* Author: Dave Coleman <davetcoleman@gmail.com>
+* Author: Alejandro Hernández
 * License: GNU General Public License, version 3 (GPL-3.0)
 
-Example robots and code for interfacing Gazebo with ROS
-
-## Tutorials
-
-[ROS URDF](http://gazebosim.org/tutorials/?tut=ros_urdf)
+Example robots and code for interfacing Gazebo, ROS2 and Moveit2
 
 ## Quick Start
 
-Rviz:
-
-    roslaunch rrbot_description rrbot_rviz.launch
-
 Gazebo:
 
-    roslaunch rrbot_gazebo rrbot_world.launch
-
-ROS Control:
-
-    roslaunch rrbot_control rrbot_control.launch
-
-Example of Moving Joints:
-
-    rostopic pub /rrbot/joint2_position_controller/command std_msgs/Float64 "data: -0.9"
```

Comment:
Isn't this a regression of functionality by removing all this? That instead should be ported rather than forgotten?

Reply:
I'd recommend at least documenting that gazebo_tutorials and rrbot_control have not yet been ported to ROS2, but need to be.

Reply:
added a note in the README.md